test/CMakeLists.txt

```diff
@@ -139,7 +145,7 @@ endif()
 
 if (NOT MSVC_STATIC_RUNTIME)
   add_fmt_executable(posix-mock-test
-    posix-mock-test.cc ../src/format.cc ${TEST_MAIN_SRC})
+    posix-mock-test.cc ${PROJECT_SOURCE_DIR}/src/format.cc ${TEST_MAIN_SRC})
```

Comment:
Das und die beiden nachfolgenden Änderungen haben nicht mit dem Module zu tun! Also bitte nicht ändern, wie alle anderen bestehenden funktionalen Test aus dem Original-Repo bei Victor.

Reply:
des `.. `gibt eine `__` im binary dir path!

Reply:
`CMake` macht das, was man in den `CMakeLists.txt` modelliert!

Die Tests sind undurchsichtig, für mich nicht nachvollziehbar modelliert?